cellprofiler/gui/pipelinecontroller.py

```diff
@@ -51,6 +51,13 @@
 import wx.lib.buttons
 import wx.lib.mixins.listctrl
 from functools import reduce
+from six import string_types, text_type
+
+try:
+    cmp             # Python 2
+except NameError:
+    def cmp(a, b):  # Python 3
```

Comment:
This is defined wherever `cmp` is needed, I think it might be good to go into `cellprofiler.utilities` (I can move it there)